vms/platformvm/api/validator.go

```diff
@@ -0,0 +1,73 @@
+// Copyright (C) 2019-2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package api
+
+import (
+	"github.com/ava-labs/avalanchego/ids"
+	"github.com/ava-labs/avalanchego/utils/json"
+	"github.com/ava-labs/avalanchego/vms/platformvm/signer"
+)
+
+// Staker is the representation of a staker sent via APIs.
```

Comment:
Is it possible to keep these definitions in the static service file?

Reply:
and later rename it as we need

Reply:
these don't belong either in static service or genesis now. It's types used by the platformvm api in `vms/platformvm/service.go` so I think we can just extract it in this PR.